.vscode/settings.json

```diff
@@ -1,8 +1,2 @@
 {
```

Comment:
@tannerlinsley This keeps being updated automatically by VS Code every time I want to contribute, is it possible maybe to ignore this?

It gets changed to:

```js
{
  "workbench.colorCustomizations": {
    "titleBar.activeBackground": "#ff4154",
    "titleBar.inactiveBackground": "#ff415499",
    "titleBar.activeForeground": "#e7e7e7",
    "titleBar.inactiveForeground": "#e7e7e799",
    "activityBar.activeBackground": "#ff7482",
    "activityBar.activeBorder": "#70ff60",
    "activityBar.background": "#ff7482",
    "activityBar.foreground": "#15202b",
    "activityBar.inactiveForeground": "#15202b99",
    "activityBarBadge.background": "#70ff60",
    "activityBarBadge.foreground": "#15202b",
    "statusBar.background": "#ff4154",
    "statusBar.border": "#ff4154",
    "statusBar.foreground": "#e7e7e7",
    "statusBarItem.hoverBackground": "#ff7482",
    "titleBar.border": "#ff4154"
  },
  "peacock.color": "#ff4154"
}
```

Reply:
Yes, you can add it to gitignore